processor/samplingprocessor/tailsamplingprocessor/processor_test.go

```diff
@@ -341,7 +438,9 @@ func generateIdsAndBatches(numIds int) ([]pdata.TraceID, []pdata.Traces) {
 			td := testdata.GenerateTraceDataOneSpan()
 			span := td.ResourceSpans().At(0).InstrumentationLibrarySpans().At(0).Spans().At(0)
 			span.SetTraceID(traceIds[i])
-			span.SetSpanID(tracetranslator.UInt64ToByteSpanID(uint64(i + 1)))
+
+			spanID++
+			span.SetSpanID(tracetranslator.UInt64ToByteSpanID(uint64(spanID)))
```

Comment:
previously this was creating duplicate span ids, now its a monotonic sequence. the other tests don't rely on the spanid it seems